src/test/resources/application.properties

```diff
@@ -1 +1,3 @@
 application.staging.url=${STAGING_URL:http://localhost:8080}
+api.user.username=${api_user_username}
```

Comment:
I assume, those are environment variables? We should keep an consistent naming scheme, so UPPERCASE please, since `STAGING_URL` is already upper case.